config.php

```diff
@@ -122,8 +122,92 @@
             'comment' => "A simple and complete solution. It speeds up processes and can eliminate the use of paper. We integrated it with our public management system or e-Cidade, it was absurdly good. Congratulations.",
             'author' => 'Igor Afonso Oliveira Ruas'
         ],
-        
-        
+    ],
+    'frequentlyQuestions' => [
+        [
+            'question' => 'Why LibreSign?',
+            'answer' => 'LibreSign allows documents to be signed securely and with legal validity, since the system generates hashing - an algorithm that ensures that the file has not been altered after being signed - as well as numbers and records the times of each signature carried out in the document. In this way, the system meets all the requirements of the GDPR - General Data Protection Law.'
+        ],
+        [
+            'question' => 'What is electronic signature capture?',
+            'answer' => 'Electronic signature capture is a technology for signing electronic document files with a handwritten signature. The use of this technology allows for the elimination of the mailing, storage, filing, copying, and retrieval of paper documents. This will save your business time and money.'
```

Comment:
Sounds a bit strange this question. I suggest to remove for now. Sounds like a question generated by an AI.